src/fiatconnect/slice.ts

```diff
@@ -147,6 +157,14 @@ export interface CreateFiatConnectTransferCompletedAction {
   txHash: string | null
 }
 
+export interface CacheFiatConnectTransferAction {
```

Comment:
this looks the same as the type you defined above. Seems like making it an alias for the other type would be more DRY